projects/igniteui-angular/src/lib/core/styles/components/navbar/_navbar-theme.scss

```diff
@@ -137,10 +137,18 @@
         }
     }
 
-    igx-action-icon {
+    igx-navbar-action,
+    [igxNavbarAction] {
         display: flex;
         align-items: center;
     }
+
+    igx-navbar-title,
+    [igxNavbarTitle] {
+        display: flex;
+        align-items: center;
+        user-select: none;
```

Comment:
I think it is better to leave the default user-select, If the users want they can set it to none in their template. @simeonoff, What do you think?

Reply:
@DiyanDimitrov @astaev I agree. Allowing text selection by default for the title seems to be the better option.